### PR TITLE
DOC-4286: ALTER INDEX documentation needs to cover partitioned index

### PR DESCRIPTION
**Pull request by simon-dew**
The following documentation is ready for review:

* [ALTER INDEX](https://simon-dew.github.io/docs-site/DOC-4286/server/6.0/n1ql/n1ql-language-reference/alterindex.html)

Documentation issue: [DOC-4286](https://issues.couchbase.com/browse/DOC-4286)

NOTE: The `replica_count` action will be documented in [DOC-5812](https://issues.couchbase.com/browse/DOC-5812).

### CODE REVIEW

File: modules/n1ql/pages/n1ql-language-reference/alterindex.adoc [hunk 1]
```diff
@@ -1,13 +1,16 @@
 = ALTER INDEX
+:page-status: Couchbase Server 5.5
```
Comment (RichardSmedley) [inline modules/n1ql/pages/n1ql-language-reference/alterindex.adoc]: Presumably the 5.5 refers to its introduction?
Having an orange badge to say so seems confusing.

File: modules/n1ql/pages/n1ql-language-reference/alterindex.adoc [hunk 2]
```diff
@@ -35,65 +39,70 @@ If a node goes down while an ALTER INDEX operation is in progress, then the inde
 Only users with the suitable RBAC role of `Administrator` are allowed to run the `ALTER INDEX` directive.
```
Comment (RichardSmedley) [inline modules/n1ql/pages/n1ql-language-reference/alterindex.adoc]: Only users with the appropriate RBAC role of `Administrator` are ...
seems more correct?
Or _necessary_ role?

**Review by RichardSmedley — APPROVED** [review]
Other than inline comments ('suitable' didn't feel right, but I'm not sure that it's wrong, _per se_), LGTM.
But I'm not an SME for this topic, of course.

File: modules/n1ql/pages/n1ql-language-reference/alterindex.adoc (hunk 1 above)
Comment (simon-dew) [reply modules/n1ql/pages/n1ql-language-reference/alterindex.adoc]: Good point, I'll keep this in for the release/5.5 backport, but remove it from release/6.0 and later.

File: modules/n1ql/pages/n1ql-language-reference/alterindex.adoc (hunk 2 above)
Comment (simon-dew) [reply modules/n1ql/pages/n1ql-language-reference/alterindex.adoc]: Removed "suitable".

**Review by deepkaran — APPROVED** [review]

**Pull request merged**